test-prow-e2e.sh

```diff
@@ -9,7 +9,8 @@ INSTALLER_DIR=${INSTALLER_DIR:=${ARTIFACT_DIR}/installer}
 function copyArtifacts {
   if [ -d "$ARTIFACT_DIR" ] && [ -d "$SCREENSHOTS_DIR" ]; then
     echo "Copying artifacts from $(pwd)..."
-    cp -r "$SCREENSHOTS_DIR" "${ARTIFACT_DIR}/integration-tests/screenshots"
+    echo "Artifacts directory... ${ARTIFACT_DIR}"
```

Comment:
nitpick: I'm not sure we need to log this since the variable will be logged above as well as the `cp -r` command in the verbose output.